[2025-07-27T02:18:22.857+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-27T02:18:22.903+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup manual__2025-07-27T02:16:42.512096+00:00 [queued]>
[2025-07-27T02:18:22.928+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup manual__2025-07-27T02:16:42.512096+00:00 [queued]>
[2025-07-27T02:18:22.929+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-27T02:18:22.963+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): cleanup> on 2025-07-27 02:16:42.512096+00:00
[2025-07-27T02:18:22.981+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=178) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-27T02:18:22.984+0000] {standard_task_runner.py:72} INFO - Started process 180 to run task
[2025-07-27T02:18:22.986+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'cleanup', 'manual__2025-07-27T02:16:42.512096+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmptoha0c2d']
[2025-07-27T02:18:22.988+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask cleanup
[2025-07-27T02:18:23.093+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.cleanup manual__2025-07-27T02:16:42.512096+00:00 [running]> on host 61bf06d32fe6
[2025-07-27T02:18:23.263+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kanewyp' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='cleanup' AIRFLOW_CTX_EXECUTION_DATE='2025-07-27T02:16:42.512096+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-27T02:16:42.512096+00:00'
[2025-07-27T02:18:23.265+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-27T02:18:23.307+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-07-27T02:18:23.309+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '\n        echo "Cleaning up resources..."\n        rm -f /app/tmp/*.pkl\n        echo "Cleanup completed."\n        ']
[2025-07-27T02:18:23.329+0000] {subprocess.py:99} INFO - Output:
[2025-07-27T02:18:23.345+0000] {subprocess.py:106} INFO - Cleaning up resources...
[2025-07-27T02:18:23.356+0000] {subprocess.py:106} INFO - Cleanup completed.
[2025-07-27T02:18:23.357+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-07-27T02:18:23.401+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-27T02:18:23.402+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=cleanup, run_id=manual__2025-07-27T02:16:42.512096+00:00, execution_date=20250727T021642, start_date=20250727T021822, end_date=20250727T021823
[2025-07-27T02:18:23.496+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-27T02:18:23.525+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-27T02:18:23.527+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
